### PR TITLE
cmd/atlas: introduce the dev-url to apply command

### PR DESCRIPTION
**Pull request by a8m**

### CODE REVIEW

File: cmd/action/apply.go [hunk 1]
```diff
@@ -31,16 +32,18 @@ var (
 	ApplyCmd = &cobra.Command{
 		Use:   "apply",
 		Short: "Apply an atlas schema to a target database.",
-		Long: "`atlas schema apply`" + ` plans and executes a database migration to bring a given database
-to the state described in the Atlas schema file. Before running the migration, Atlas will print the migration
-plan and prompt the user for approval.
+		// Use 80-columns as max width.
```
Comment (masseelch) [inline cmd/action/apply.go:35]: :-1: I don't like 80 cols. We are no longer working on SD resolutions and no-one is coding on a smartphone

**Review by masseelch — APPROVED** [review]

File: cmd/action/apply.go (hunk 1 above)
Comment (a8m) [reply cmd/action/apply.go:35]: What do you don't like about this output? When the screen size is smaller (my case), the line breaks make it ugly. 

image

Comment (masseelch) [reply cmd/action/apply.go:35]: Your screen supports 120 too by the looks of it.

Comment (masseelch) [reply cmd/action/apply.go:35]: image

**Pull request merged**